include/fg/CPUCopy.hpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************
```

Comment:
but these functions are under fg namespace and it makes sense to leave them under fg folder.